src/dev/flang/be/jvm/JVM.java

```diff
@@ -994,10 +997,9 @@ else if (r == -1)   // a constructor
 
         */
 
-        var jt = _types.javaType(t);
         var ft = _types.resultType(t);
         var getf =
-          fieldExists(r) ? (Expr.aload(current_index(cl), jt)
+          fieldExists(r) ? (Expr.aload(current_index(cl), ft, new VerificationType(VerificationType.type.Object, 2 /* NYI magic constant... */) )
```

Comment:
no magic please. Might be solved by creating cpool entries later, see other comment.